examples/knative/powercli/kn-pcli-telegram/README.md

```diff
@@ -0,0 +1,149 @@
+# kn-pcli-telegram
+Example Knative PowerCLI function for sending a [Telegram](https://telegram.org/) message when a VM is migrated and also resides in a specific vSphere Resource Pool.
```

Comment:
I understand the script will also work if VM is not part of RP right?

If so, suggesting to reword:

```suggestion
Example Knative PowerCLI function for sending a [Telegram](https://telegram.org/) message when a VM is migrated and optionally filter by its location in a vSphere Resource Pool.
```